vcpkg/get_started/get-started-linux.md

```diff
@@ -0,0 +1,129 @@
+ms.date: 08/23/2023
+ms.prod: vcpkg
+#CustomerIntent: As a beginner C++ developer, I want to learn how to install and manage packages on a Linux environment using vcpkg, so that I can easily set up and maintain C++ projects with necessary dependencies.
+---
+
+# Tutorial: Install and use packages on Linux with vcpkg
+
+This tutorial shows how to set up a C++ "Hello World" CMake project using the `fmt` library. Manage dependencies, construct the application, and execute the project with the help of vcpkg. This guide is tailored for a Linux environment.
+
+## Prerequisites
+
+- A terminal to work in.
+- `git`, `CMake`, and a C++ compiler installed on your system
+- vcpkg installed on your system
+
+## 1 - Set up the project
+
+1. Create the project directory
+
+    This tutorial assumes your project is located in `/home/helloworld`. If you intend to place your project somewhere else, replace any references to that path with your preferred path.
+
+    Create a directory named "helloworld" to contain your project files, put it right next to your vcpkg directory. Then, navigate to your project directory.
+
+    ```bash
+    mkdir helloworld && cd helloworld
+    ```
+
+2. Create the manifest
+
+    Next, create a file named `vcpkg.json` in your project's directory, this is your manifest file. vcpkg reads the manifest file to learn what dependencies to install.
+
+    From within the `helloworld` directory, run:
+
+    ```bash
+    ..\vcpkg\vcpkg new --application
+    ```
+
+    The `vcpkg new` command adds a `vcpkg.json` file and a `vcpkg-configuration.json` file to your projects directory. The `vcpkg.json` file should look like this:
+
+    ```json
+    {}
+    ```
+
+    The `vcpkg.json` file serves as the foundation to integrate vcpkg in your C++ project. Initially, the file contains an empty JSON object; additional attributes like dependencies and supported features can be added as your project grows.
+
+    The `vcpkg-configuration.json` file serves a different purpose. This file allows full control over your dependencies' sources and it's out of scope for this tutorial.
+
+    For this tutorial, all the dependencies come from the vcpkg registry at <https://github.com/Microsoft/vcpkg>. When there's no explicit configuration provided, vcpkg uses this registry as default.
+
+## 2 - Add dependencies and project files
+
+Now that the project is set up, add the `fmt` library as a dependency and generate the project files.
+
+1. Add `fmt` dependency
+
+    To do so, run:
+
+    ```bash
+    ..\vcpkg\vcpkg add port fmt
+    ```
+
+    Your `vcpkg.json` should look like this:
+
+    ```json
+    {
+        "dependencies": [
+        "fmt"
+        ]
+    }
+    ```
+
+2. Create project files
+
+    Create the `CMakeLists.txt` and `main.cpp` files:
+
+    ```bash
+    touch CMakeLists.txt main.cpp
+    ```
+
+    Fill in `CMakeLists.txt` with the necessary information to find and link the `fmt` package. The `CMakeLists.txt` should look like this:
```

Comment:
(opinion) I'm not sure I like the way this first sentence is presented.

> Fill in `CMakeLists.txt` with the necessary information to find and link the `fmt` package.

As a reader, I feel like this is an instruction that implies I should know how to complete it without reading further. If I was a new user, it would cause me some confusion ("I don't know how to find and link `fmt`; you told me to use it!") until I read further.

Instead, perhaps we should simplify the instruction to "Create a CMakeLists.txt file with the following contents.", and then explain what the file does.

Reply:
agree